packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/key-message.twig

```diff
@@ -39,45 +43,45 @@
 
   var id = "{{ id }}";
   var blckid = "blck{{ id }}";
-  var halfImage = `{{ keyMessage.halfImage }}` // this returns '1' for true, '' or '0' for false or undefined. 
+  var halfImage = `{{ keyMessage.halfImage }}`; // '1' for true, '' or '0' for false/undefined
 
   function marginSet() {
     var block = document.getElementById(blckid);
     var mainContent = document.getElementById("main-content");
-    var keyMessageSection = document.getElementById(id)
+    // Check if within a microsite
+    var keyMessageSection = document.getElementById(id);
+    var keyMessageExtra = document.querySelector('.ma__microsite-key-message');
```

Comment:
Is this going to be necessary after reverting the top positioning? If so, can we do this in openmass using CSS instead? we are trying not to avoid referencing class names that are controlled in openness only